### PR TITLE
Feature/release

### PR DESCRIPTION
**Pull request by iruzevic**
### Update

- Boilerplate is clean and is using only temporary files used for the initial setup. After running the setup, the temporary files are removed and replaced with the actual files.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,16 +22,10 @@
...
-		"infinum/eightshift-coding-standards": "2.0.0-beta",
-		"php-stubs/wordpress-stubs": "^6.3",
-		"szepeviktor/phpstan-wordpress": "^1.3",
 		"wp-cli/wp-cli": "^2.9"
 	},
 	"require": {
-		"php": "^7.4 || >=8.0",
-		"infinum/eightshift-libs": "^7.0.0",
-		"typisttech/imposter-plugin": "^0.6.2"
+		"php": "^7.4 || >=8.0"
```
Comment (goranalkovic-infinum) [inline composer.json:28]:
```suggestion
		"php": ">=8.0"
```
As this is a very major new version, I'd say we're very fine to ditch 7.4.
I'd even consider making 8.2 the min version.

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (iobrado) [reply composer.json:28]: ☝️

**Review by iobrado — APPROVED** [review]

**Review by piqusy — DISMISSED** [review]
Junior dev: writes 1000 lines of code a day
Senio dev: writes 50 lines of code  a day
10x dev: deletes 1000 lines of code a day

File: composer.json (hunk 1 above)
Comment (iruzevic) [reply composer.json]: done

**Comment by goranalkovic-infinum** [comment]
> Junior dev: writes 1000 lines of code a day Senio dev: writes 50 lines of code a day 10x dev: deletes 1000 lines of code a day

Bica: updates the code

**Review by goranalkovic-infinum — APPROVED** [review]

**Review by iobrado — APPROVED** [review]

**Pull request merged**